yt_dlp/downloader/external.py

```diff
@@ -269,18 +274,17 @@ def _make_cmd(self, tmpfilename, info_dict):
         cmd += self._bool_option('--check-certificate', 'nocheckcertificate', 'false', 'true', '=')
         cmd += self._bool_option('--remote-time', 'updatetime', 'true', 'false', '=')
         cmd += ['--auto-file-renaming=false']
+
         if 'fragments' in info_dict:
-            cmd += verbose_level_args
-            cmd += ['--uri-selector', 'inorder', '--download-result=hide']
```

Comment:
all these are needed. The options given in `self._configuration_args` can be overridden by user but these can't